index.html

```diff
@@ -0,0 +1,81 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet noopener noreferrer" href="css/style.css" />
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <nav id="navbar">
+      <ul class="nav navbar-nav">
+        <li class="active"><a href="#">Home</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Work</a></li>
+        <li><a href="#">Gallery</a></li>
+      </ul>
+    </nav>
+    <section id="welcome-section">
+      <h1>Hello welcome to our portfolio</h1>
+    </section>
+    <section id="projects">
+      <div>
+        <div id="project-header"><h4>Checkout our Awesome Projects</h4></div>
+        <div class="some-page-wrapper">
+          <div class="row">
+            <div class="column">
+              <div class="orange-column">
+                <div class="project-tile">
+                  <a href="www.ourproject.com">Project 1</a>
```

Comment:
- [ ] Nice to have a class attribute assigned to an anchor tag `<a>`. Doing so makes it easier to style the links.